targets/tools/js2c.py

```diff
@@ -21,8 +21,12 @@
 import os
 import re
 
+import argparse
+
+special_chars = re.compile('[-|\\\\|\\?|\\\'|".]')
```

Comment:
This regex does not seem right. Within character classes, you don't need alternation (actually, special characters lose their meaning). And using raw strings could help to get rid of lots of backspace escaping. A quick try in python shell gave:
```py
>>> special_chars = re.compile(r'[-\\?\'".]')
>>> special_chars.sub('_', "x-x?x'x\"x.x\\x")
'x_x_x_x_x_x_x'
```
(If you want to replace dash, backslash, question mark, single and double quotes, and period.)